xbmc/addons/settings/AddonSettings.cpp

```diff
@@ -39,6 +39,9 @@
 #include <mutex>
 #include <vector>
 
+namespace
+{
+
 static const std::string OldSettingValuesSeparator = "|";
 
 static const int UnknownSettingLabelIdStart = 100000;
```

Comment:
not sure if an anonymous namespace + static makes sense here. so i would change this to
```cpp
namespace
{
constexpr auto OldSettingValuesSeparator = "|";
constexpr int UnknownSettingLabelIdStart = 100000;
```

Reply:
Have changed to constexpr.